### ISSUE TITLE
Screen reader announces "Enter to Refactor, Ctrl+Enter to Preview" for every refactoring

### ISSUE THREAD

**Issue by leonardder** [microsoft/vscode#157402]
Type: <b>Bug</b>

Since I'm using the most recent insiders build, I'm getting this message for every suggested refactoring after pressing Ctrl.period at least when using Rust. Note that the actual text of the refactoring is still accessible by using NVDA object navigation, but it isn't read when using up and down arrows

VS Code version: Code - Insiders 1.71.0-insider (73fd3f11032e7b83c2ae011b5516e6ddd19e3db2, 2022-08-05T07:36:58.753Z)
OS version: Windows_NT x64 10.0.25151
Modes:
Sandboxed: No

cc @isidorn

**Comment by isidorn** [comment]
Sounds like a bug to me / regression to me. Let me assign it to August so we do not miss this. 

@justschen the new refactoring UI is probably not accessible. If you are using a list make sure to properly use the `accessibilityProvider` interface. Also feel free to reach out to me and we can sync for 30 mins on accessibility.

@leonardder thanks for reporting!

**Comment by jvesouza** [comment]
Confirmed on my fedora version 36 workstation, using orca.

**Comment by justschen** [comment]
Taking a look at how we're doing hovers ATM! We're using the native hover (ie, using html titles), but will take a look at the `accesibilityProvider` that Isidor mentioned!

**Comment by jvesouza** [comment]
Works OK on my fedora workstation. Great job.

**Comment by isidorn** [comment]
Thanks for letting us know. Adding verified label.

**Comment by leonardder** [comment]
I was also able to verify that it works again. There is a difference though, the instruction to press either enter or ctrl-enter is in the accDescription (likely aria-description or aria-describedby) of every suggestion. Even when it is possible to silence it, I find it slightly noisy.
